src/librbd/migration/HttpClient.h

```diff
@@ -110,7 +109,7 @@ class HttpClient {
       }
 
       void operator()(
-          boost::beast::ssl_stream<boost::beast::tcp_stream>& stream) override {
+	boost::asio::ssl::stream<boost::asio::ip::tcp::socket>& stream) override {
```

Comment:
Nit: Preserve previous indentation -- it has the benefit of being different from the method body:
```suggestion
	  boost::asio::ssl::stream<boost::asio::ip::tcp::socket>& stream) override {
```